src/pages/ProposalsSpam.jsx

```diff
@@ -0,0 +1,221 @@
+import React, { useEffect, useState, Fragment } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import {
+  Box,
+  Table,
+  Thead,
+  Tr,
+  Th,
+  Tbody,
+  Td,
+  IconButton,
+  Flex,
+} from '@chakra-ui/react';
+import { RiRefreshLine } from 'react-icons/ri';
+
+import { useTX } from '../contexts/TXContext';
+import useBoost from '../hooks/useBoost';
+import MainViewLayout from '../components/mainViewLayout';
+import ContentBox from '../components/ContentBox';
+import ListSelect from '../components/listSelect';
+import { timeToNow } from '../utils/general';
+import { fetchAllActivity } from '../utils/theGraph';
+import { SPAM_FILTER_UNSPONSORED } from '../graphQL/dao-queries';
+import { proposalResolver } from '../utils/resolvers';
+
+const FILTERS = {
+  main: [
+    { name: 'All', value: 'all' },
+    { name: 'Unsponsored', value: 'Unsponsored' },
+    { name: 'Cancelled', value: 'Cancelled' },
+  ],
+};
+
+const proposalRow = (proposal, daochain, daoid) => {
```

Comment:
Since this is a component, should probably change this to pascal case. ProposalRow

Reply:
Oh, it's not being used as component. 

I'd say since it's pretty much a component, we could probably turn it into one. If should be a function than ignore these comments.

Reply:
:heavy_check_mark: